app/containers/MessageActions/index.tsx

```diff
@@ -237,6 +239,23 @@ const MessageActions = React.memo(
 				replyInit(message, false);
 			};
 
+			const handleReplyInDM = async (message: TAnyMessageModel) => {
+				if (message.u.username) {
+					const result = await Services.createDirectMessage(message.u.username);
+					if (result.success) {
+						const { room } = result;
+						const params = {
+							rid: room.rid,
+							name: getRoomTitle(room),
+							t: room.t,
+							roomUserId: getUidDirectMessage(room),
+							replyInDM: message
+						};
+						Navigation.replace('RoomView', params);
```

Comment:
We should try to use `goRoom`

Reply:
The `goRoom` uses the `Navigation.navigate`, this navigation's type doesn't work when it goes to the "same" view. Also, this handleInDM introduces that behavior that we commented on some time ago, about the navigation between chats.

Reply:
`goRoom` accepts a `navigationMethod` param.
However remember we're going to deprecate nested rooms soon and we'll have to analyze this case again to use `goRoom` anyway.